guides/manual-installation.md

```diff
@@ -391,6 +392,7 @@ journalctl -fu trento-web
     --add-host "host.docker.internal:host-gateway" \
     -e AMQP_URL=amqp://trento_user:trento_user_password@host.docker.internal/vhost \
     -e ENABLE_ALERTING=false \
+    -e ENABLE_OIDC=false  \
```

Comment:
The same, you don't really need to false, specially as you document later on how to enable it